tests/test_clinical.py

```diff
@@ -956,8 +956,15 @@ def test_remap_clinical_values_sampletype():
 )
 def test_remap_clinical_values(col):
     """Test Remapping clinical values"""
-    testdf = pd.DataFrame({col: [1, 2, 99]})
-    expecteddf = pd.DataFrame({col: ["Male", "Female", "Unknown"]})
+    testdf = pd.DataFrame({"SEX": [1, 2, 99], "PRIMARY_RACE": [1, 2, 99]})
```

Comment:
I would add a couple more tests here:

- test for having all code columns (to make the detailed cols) you could have so we make sure it remaps for all of them
- test for when you have none of the columns

Reply:
Done.